utils/provided-goi-analysis-functions.R

```diff
@@ -184,21 +183,57 @@ plot_goi_expression_sina <- function(normalized_sce,
         )
       )
     
+    # calculate average gene expression
+    avg_gene_exp = mean(colMeans(logcounts(normalized_sce)))
+    
     sina_expression_plot <- ggplot(expression_means_df, aes(
       x = !!gene_symbol_column_sym,
       y = gene_expression,
       color = gene_expression)) +
-      ggforce::geom_sina() +
-      theme(axis.text.x = element_text(angle = 90))
+      ggforce::geom_sina(size = 0.2) +
+      theme(axis.text.x = element_text(angle = 90)) +
+      stat_summary(
+        aes(group = !!gene_symbol_column_sym),
+        color = "red",
+        # median and quartiles for point range
+        fun = "mean",
+        fun.min = function(x) {
+          quantile(x, 0.25)
+        },
+        fun.max = function(x) {
+          quantile(x, 0.75)
+        },
+        geom = "pointrange",
+        position = position_dodge(width = 0.7),
+        size = 0.2,
+        shape = 21
+      ) +
+      geom_hline(yintercept = avg_gene_exp, type = "dashed")
```

Comment:
```suggestion
      geom_hline(yintercept = avg_gene_exp, linetype = "dashed")
```
This argument should be `linetype` to actually make it dashed.